lib/cyclone-server/src/handlers.rs

```diff
@@ -100,155 +96,120 @@ pub async fn ws_execute_ping(
 
 pub async fn ws_execute_resolver(
     wsu: WebSocketUpgrade,
-    State(lang_server_path): State<LangServerPath>,
-    State(telemetry_level): State<TelemetryLevel>,
-    State(lang_server_function_timeout): State<LangServerFunctionTimeout>,
     State(lang_server_process_timeout): State<LangServerProcessTimeout>,
+    State(child): State<LangServerChild>,
```

Comment:
Oh yeah, wow, that is different--the process is already running and we only need to provide a handle to the child process.